### DISCUSSION TITLE
Command/economy on spawner break/place/change

### DISCUSSION THREAD

**Discussion by EkonC** [Zrips/CMI#4797] [Enhancement]
Hello, I think that you should be able to perform commands or at least charge money when the player changes a spawner, breaks a spawner or places a spawner. Really great would be if you could get a confirming message before doing so.

**Comment by winnipeg21** [comment]
Basically features from mineablespawners. Being able to charge prices for breaking spawners (default price for all) then set prices per spawner too. Also, add the ability to run commands when broken, Solid idea
